### PR TITLE
Tejal group code

### PR DESCRIPTION
**Pull request by SEjss**
Pull Request for Tejal Group code branch

### CODE REVIEW

File: src/test/resources/features/groupcode_tejal.feature [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  Feature: Registration Scenario
+
+
+Background:
+Given I OPEN  "registration" page
+ And I type "Kavya" into firstName field
+  Then I type "Shah" into lastName field
+  Then I type "tss@gmail.com" into email input field
+  Then I type "JUL22" into groupCode field
+  Then I type "1981" into password field field
```
Comment (gpl-git) [inline src/test/resources/features/groupcode_tejal.feature:11]: On lines 11 and 12 you need at least 5 characters input

File: src/test/resources/features/groupcode_tejal.feature [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    Then Error message should be appeared
+    Then Error Message "Should no more than 10 characters" should be displayed
+
+    @registration_GroupCode3
+    Scenario: GroupCode Allows min 1 char
+      Given I again clear on groupCode field
+      Then I type "s" into  groupCode space
+      Then  I clear on groupCode field again
+      Then I click Register-me button
+      And I get an error message
```
Comment (gpl-git) [inline src/test/resources/features/groupcode_tejal.feature:29]: Please update according to the application behavior

File: src/test/resources/features/groupcode_tejal.feature (hunk 1 above)
Comment (SEjss) [reply src/test/resources/features/groupcode_tejal.feature:11]: Scenario: Group Code cant keep empty
expected Result: Group code Field can not be empty. error message "Field Can't be Empty" should be displayed
Actual Result: Error message doesn't displayed and user can register the page.
Bug report: https://jira.portnov.com/browse/JUL22-832 

Scenario:  User can register when 'Group Code' has white spaces
Expected Result: User can not register if there is a white space between the word and Error message" white space is not allowed" should be displayed

Actual Result: Error message doesn't display and user can Register the page
Bug Report :https://jira.portnov.com/browse/JUL22-360

File: src/test/resources/features/groupcode_tejal.feature (hunk 2 above)
Comment (SEjss) [reply src/test/resources/features/groupcode_tejal.feature:29]: Scenario: Group Code cant keep empty
expected Result: Group code Field can not be empty. error message "Field Can't be Empty" should be displayed
Actual Result: Error message doesn't displayed and user can register the page.
Bug report: https://jira.portnov.com/browse/JUL22-832

Scenario: User can register when 'Group Code' has white spaces
Expected Result: User can not register if there is a white space between the word and Error message" white space is not allowed" should be displayed

Actual Result: Error message doesn't display and user can Register the page
Bug Report :https://jira.portnov.com/browse/JUL22-360

**Pull request closed without merging**